python/pyspark/mllib/clustering.py

```diff
@@ -90,6 +90,14 @@ class KMeansModel(Saveable, Loader):
     ...     rmtree(path)
     ... except OSError:
     ...     pass
+
+    >>> data = array([-383.1, 31.2, 32.2, 367.3]).reshape(2, 2)
```

Comment:
I would recommend keeping the original data structure; I could imagine us later adding checks which print warnings when given 2 data points but 3 cluster centers.